interface/patient_file/report/patient_report.php

```diff
@@ -40,12 +40,12 @@
 $auth_med      = AclMain::aclCheckCore('patients', 'med');
 $auth_demo     = AclMain::aclCheckCore('patients', 'demo');
 
-$oefax = !empty($GLOBALS['oefax_enable']) ? $GLOBALS['oefax_enable'] : 0;
 /**
  * @var EventDispatcherInterface $eventDispatcher  The event dispatcher / listener object
  */
 $eventDispatcher = $GLOBALS['kernel']->getEventDispatcher();
 ?>
+<!DOCTYPE>
```

Comment:
hi @juggernautsei , just noted this `<!DOCTYPE>` here. Why was this added?

Reply:
I found a couple other places where this is used in codebase, but they contain html `<!DOCTYPE html>`

Reply:
I installed a new tool for me in my PHPStorm. It is called SonarLint. It made a lot of code compliance suggestions. I only made a few of those changes in this pull request.